### PR TITLE
custom domains

### PR DESCRIPTION
**Pull request by saggineumann**

### CODE REVIEW

File: docs/how-to-customize-my-service/setting-up-a-custom-sftp-server-name.md [hunk 1]
```diff
@@ -27,3 +28,6 @@ Here are some links to the documentation regarding editing DNS records with popu
 | Wordpress | [Adding Custom DNS Records](https://wordpress.com/support/domains/custom-dns/#adding-custom-dns-records) |
 | Wix | [Adding or Updating CNAME Records](https://support.wix.com/en/article/adding-or-updating-cname-records-in-your-wix-account) |
 
+:::tip
+Adding a custom domain for your SFTP endpoint is available in all plans. However, if you'd like to change the domain that shows as default in SFTP To Go's admin dashboard, read more about this option [here](../getting-started/organization-settings#domains). This feature is only available with [certain plans](https://sftptogo.com/pricing).
```
Comment (motymichaely) [inline docs/how-to-customize-my-service/setting-up-a-custom-sftp-server-name.md:32]: @saggineumann The ability to set a custom domain for SFTP / FTPS endpoints through the Domains settings is available for all plans. Only web portal domains is restricted for certain plans.

File: docs/getting-started/organization-settings.md [hunk 2]
```diff
@@ -23,7 +23,30 @@ Use the audit logs to monitor access to your storage. The Audit logs dialog lets
 
 Set your organization's password policy for credentials. This will not change existing passwords, but changes to passwords (either ones generated automatically or manually) must adhere to the organization's current password policy.
 
-## Inbound network rules
+## Network
+
+### Domains
+
+Custom domains allow you to customize and brand your SFTP To Go SFTP endpoints for both SFTP and web portal access.
```
Comment (motymichaely) [inline docs/getting-started/organization-settings.md:30]:
```suggestion
Custom domains allow you to customize and brand your SFTP To Go endpoints for both SFTP / FTPS and web portal access.
```

File: docs/getting-started/organization-settings.md [hunk 3]
```diff
@@ -23,7 +23,30 @@ Use the audit logs to monitor access to your storage. The Audit logs dialog lets
...
+
+To add a new custom domain:
+
+1. Click **Add domain**.
+2. Enter the fully qualified domain you'd like to use as an endpoint. Note that root domains cannot be used (i.e., use files.example.io instead of example.io).
+3. Select the type of endpoint that the domain will be used for:
+    * SFTP/FTPS - For use with FTP clients. Requires adding a CNAME pointing to the endpoint.
+    * Web portal - Accessible with any web browser. Requires adding a CNAME pointing to the endpoint. TLS certificates are managed by SFTP To Go.
+4. Click **Add domain** to return to the list.
+5. Wait for instructions on what records to add in your DNS manager. Either apply them yourself or send them to your network manager.
```
Comment (motymichaely) [inline docs/getting-started/organization-settings.md:40]:
```suggestion
5. Wait for instructions on what records to add in your DNS provider. Either apply them yourself or send them to your network manager.
```

File: docs/getting-started/organization-settings.md [hunk 4]
```diff
@@ -23,7 +23,30 @@ Use the audit logs to monitor access to your storage. The Audit logs dialog lets
...
+6. It may take some time for DNS records to apply. You can wait for the state of your domain to show as "Verified" or click the ... menu button to manually refresh the state.
+7. Once the domain has been verified, you can click the ... menu button to set it as the default domain, which will replace the default domain with the newly set up domain in the SFTP To Go admin dashboard.
+
```
Comment (motymichaely) [inline docs/getting-started/organization-settings.md:43]: @saggineumann I think we should add the table we have here (https://github.com/crazyantlabs/sftptogo-docs/pull/164/files#diff-2882505e068f0a6c0d51657d31d153733bc59c664ce932c64faedba832160e59R15) to the list of domain providers and instructions on setting up DNS records.

**Review by motymichaely — CHANGES_REQUESTED** [review]
Added a few comments

File: docs/how-to-customize-my-service/setting-up-a-custom-sftp-server-name.md (hunk 1 above)
Comment (saggineumann) [reply docs/how-to-customize-my-service/setting-up-a-custom-sftp-server-name.md:32]: good to know! :) i'll fix it.

File: docs/getting-started/organization-settings.md (hunk 4 above)
Comment (saggineumann) [reply docs/getting-started/organization-settings.md:43]: sure, will do.

**Comment by saggineumann** [comment]
@motymichaely i've made the changes

**Review by motymichaely — APPROVED** [review]

**Pull request merged**